### PR TITLE
blog/themes/3mdeb: start aligning menu with wp theme from 3mdeb.com

### PR DESCRIPTION
**Pull request by pietrushnic**
Signed-off-by: Piotr Król <piotr.krol@3mdeb.com>

### CODE REVIEW

**Comment by pietrushnic** [comment]
@artur-rs @macpijan when we can merge it? Do you see any reason why not to do that? Maybe first to develop?

**Comment by artur-rs** [comment]
@pietrushnic problems I've found on the Chrome Version 98.0.4758.102 (Official Build) (64-bit):
1. On localhost below 1102px width, there is no icon for hamburger menu
2. >= 1102 px the menu is cropped (contact is not fully shown)
3. >= 1441 px 3mdeb logo is clipped to the left border, without any indent

Is it my local setup or you can reproduce my issues?

**Comment by pietrushnic** [comment]
@artur-rs key question is if what you see is new behavior or also visible on old code, if this is new I will look how to fix that, if old then I would not care much.

**Comment by artur-rs** [comment]
I've made review based on 3mdeb.com menu/header as a referrence. Imo it's a new bug

**Comment by pietrushnic** [comment]
@artur-rs point 2 is not a regression since current version on blog.3mdeb.com also does not support that, this maybe more related to Hugo then to theme, but anyway. My main question is what those tests prove? Is it a concern with support on certain resolution, some device or general concern about RWD?

File: blog/themes/3mdeb/static/css/blog.css [hunk 1]
```diff
@@ -136,41 +138,166 @@ body {
...
+    padding:0 0px 0 0;
+    border-bottom: 2px solid transparent;
+  	width: 100%;
   }
 
+.bigmenu .mainMenu a{
+    padding-left: 0;
+}
+  
+  /*.bigmenu li a:hover {
```
[the remark this answers is not in the document]
Comment (pietrushnic) [reply blog/themes/3mdeb/static/css/blog.css:233]: No, git will remember.

File: blog/themes/3mdeb/static/css/bulma.css [hunk 2]
```diff
@@ -11019,9 +11028,38 @@ img.social-icon {
 }
 
 @media (max-width: 1024px) {
+
   .right-row-wrapper{
     flex: 0 0 65%;
   }
+
+  /*.header .inner-header .menu-icon {
```
[the remark this answers is not in the document]
Comment (pietrushnic) [reply blog/themes/3mdeb/static/css/bulma.css:11036]: No, git will remember.

File: blog/themes/3mdeb/static/css/blog.css [hunk 3]
```diff
@@ -14,41 +14,42 @@ body {
...
 .header li a:hover,
 .header .menu-btn:hover {
-  color: #38d430;
+  /*color: #38d430;*/
 }
 
+
+
+
+
```
[the remark this answers is not in the document]
Comment (pietrushnic) [reply blog/themes/3mdeb/static/css/blog.css]: removed

**Review by artur-rs — APPROVED** [review]

**Pull request merged**